### DISCUSSION TITLE
Presoldered Outdoor

### DISCUSSION THREAD

**Discussion by stephenjamieson** [ajfriesen/ESPHome-AirGradient#14] [Ideas]
Now that airgradient is selling an [outdoor unit](https://www.airgradient.com/open-airgradient/outdoor/), thoughts on how difficult it would be to integrate?  Looks like this one is using an esp32.

**Comment by ajfriesen** [comment]
Should be doable.
I found a schematic from the docs.
If you have a unit I would use this to figure out which component is at which pin.

https://www.airgradient.com/open-airgradient/instructions/diy-open-air-documentation-v11/

**Reply by stephenjamieson** [reply]
Yes, I have a unit.  Happy to provide anything I can

**Reply by ajfriesen** [reply]
You basically take the same yaml and then change the pins depending if the sensors are connected elsewhere.

Oh and of course change the board to esp32.

I did not check if this device uses other sensors, so you might need to check this as well.

If you go one sensor at the time you should be able to figure this out.
